app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<p>Mechanic: <%= @mechanic.name %></p>
+<p>Years of Experience: <%= @mechanic.years_experience%></p>
+
+<p>Current rides they're working on:</p>
+<% @mechanic.rides.each do |ride| %>
+  <p><%= ride.name %></p>
+<% end %>
+
+<p>Add a ride to workload:</p>
+<%= form_with url: "/mechanics/#{@mechanic.id}", method: :patch, data: { turbo: false } do |form| %>
```

Comment:
Okay so with your form, to make it more RESTful, we will want our path to be essentially what we are doing. What are we doing? We already have a mechanic, we already have a ride, so what we are doing is creating that new association between the two - `RideMechanic`. So we would want our `url` to look like:
```ruby
url: "/mechanics/#{@mechanic.id}/rides"
```
Because we are `creating` a new association, we will want a method of `post`. So your route would be: 
```ruby
post "/mechanics/:id/rides", to: "ride_mechanic#create"
```
Then you would need to create a `ride_mechanic_controller` and a `create` action. So the create action might look like:
```ruby
def create
  RideMechanic.create(mechanic_id: params[:id], ride_id: params[:ride_id])
  redirect_to "/mechanics/#{params[:id]}"
end
```